include/unkstruct.h

```diff
@@ -314,4 +314,10 @@ typedef struct {
     /* 0x4 */ u32 unk4;
     /* 0x8 */ s32 damageTaken;
     /* 0xC */ s32 unkC;
-} Unkstruct_800FE97C;
\ No newline at end of file
+} Unkstruct_800FE97C;
+typedef struct {
+    s32 unk0;
+} Unkstruct_oneWord;
+typedef struct {
+    u32 unk0;
+} Unkstruct_oneUword;
```

Comment:
You can probably use u32 and s32 arrays for these until we find the right structs